### DISCUSSION TITLE
Cannot access to the Documents

### DISCUSSION THREAD

**Discussion by ZhHe11** [PKU-Alignment/safety-gymnasium#111] [Q&A]
Cannot access to the Documents(https://www.safety-gymnasium.com/).

**Comment by muchvo** [comment]
[This discussion](https://github.com/PKU-Alignment/safety-gymnasium/discussions/104) may help you.

**Reply by ZhHe11** [reply]
Sorry, I didn't it before. Thanks a lot!
